Packs/CommonScripts/Scripts/CheckIndicatorValue/CheckIndicatorValue.py

```diff
@@ -0,0 +1,60 @@
+import base64
+import urllib.parse
+from typing import Any, Dict, Optional
+
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+
+def exists_indicator(indicator: str) -> bool:
+    if contents := execute_command('getIndicator', {'value': indicator}):
+        if len(contents) > 0 and contents[0].get('value') in [indicator]:
+            return True
+    return False
+
+
+def decode_indicator(indicator: str, encoding: Optional[str]) -> str:
+    if not encoding or encoding == 'none':
+        return indicator
+    elif encoding == 'base64':
+        return base64.b64decode(indicator.encode()).decode()
+    elif encoding == 'url-encoding':
+        return urllib.parse.unquote(indicator)
```

Comment:
How should the customers know which encoding to use?
Also, please use our formatters, as it might cause bugs when we save something one way, but look it up another way.
You have FormatURL Script for urls, for example.

Reply:
The demo video is recored in the Contribution Registration form.